src/carousel.tsx

```diff
@@ -73,6 +73,70 @@ export const PageBullets: React.FC<PageBulletsProps> = ({currentIndex, numPages,
     );
 };
 
+const defaultGoPrev = () => {
+    throw new Error('You must wrap your component with a CarouselContextProvider to use CarouselContext');
+};
+const defaultGoNext = defaultGoPrev;
+const defaultBulletProps = {currentIndex: 0, numPages: 0};
+
+type CarouselControls = {
+    goPrev: () => void;
+    goNext: () => void;
+    bulletsProps: PageBulletsProps;
+};
+
+const CarouselContext = React.createContext<CarouselControls>({
```

Comment:
it would be amazing if we could goToPage(X). 

In the CMS we could use that to adjust the preview to the page the operator is currently editing.

Reply:
done